src/Microsoft.DotNet.Cli.Utils/Constants.cs

```diff
@@ -20,6 +20,7 @@ internal static class Constants
 
         public static readonly string HostExecutableName = "corehost" + ExeSuffix;
         public static readonly string DefaultConfiguration = "Debug";
+        public static readonly string DefaultPublishConfiguration = "Release";
```

Comment:
I will switch default to Debug but I think will leave it as separate constant since IMO this should be Release in the future. What do you think about that?